infra-in-containers/resources/docker-compose.yml

```diff
@@ -0,0 +1,122 @@
+version: '2'
+services:
+  rainloop:
+    container_name: rainloop
+    image: hardware/rainloop
+    links:
```

Comment:
I believe `links` is deprecated and no longer needed. Please verify.

Reply:
verified and removed